### DISCUSSION TITLE
Can Robyn enable Geo or segment level analysis?

### DISCUSSION THREAD

**Discussion by DongHarry-Kang** [facebookexperimental/Robyn#856] [Q&A] [answered]
Hi all -

I am currently learning Robyn and Lightweight MMM. I noticed that Lightweight MMM allows Geo labels which can provide segment (market as different geo label) level ROI and sales contribution per channel. However, I do not see such function in Robyn. is there anything we can do with Robyn if I want to know ROI and sales contribution per channel in different segments?

**Comment by SeanRichterWalsh** [comment, marked as answer]
Not in the same way as LightweightMMM. I think you would have to split your data by geo and then build a Robyn MMM for each subset.
